.github/workflows/test.yml

```diff
@@ -284,9 +285,12 @@ jobs:
       - name: Display Python version
         run: python -c "import sys; print(sys.version)"
 
+      - name: Install OSX packages
+        run: ./scripts/install_osx_dependencies.sh
```

Comment:
Line 289 uses `./scripts`.

Reply:
that's probably it! The correct one is for the workflow that does not break in the diagnostic PR, I reckon...

Reply:
nice catch. I couldn't figure this one out 🤣